### ISSUE TITLE
Documentation: upgrade guide from v0 to v1

### ISSUE THREAD

**Issue by NoryDev** [basecamp/google_sign_in#19]
# About

Following up on https://github.com/basecamp/google_sign_in/pull/18, I think an upgrade guide from v0 to v1 could be valuable. 

I get that you might not think it's useful since in v0 we (gem users) should not have expected stability, but if you still think it's valuable I'd gladly provide a PR.

In that case, please advice if it's better as a wiki entry or as a file in the repo.

**Comment by NoryDev** [comment]
@georgeclaghorn I don't think we can do a PR for a wiki entry in github. I updated the wiki in my fork with what I had in mind: 

the page: https://github.com/NoryDev/google_sign_in/wiki/Upgrade-from-V0x-to-V1x
the wiki repo: https://github.com/NoryDev/google_sign_in.wiki.git

I'm not sure how to proceed from this point on.

**Comment by georgeclaghorn** [comment]
That’s a good start! Feel free to copy it to this repository.

**Comment by NoryDev** [comment]
done!
